radix-common/src/types/addresses/component_address.rs

```diff
@@ -33,6 +33,10 @@ impl ComponentAddress {
         self.0.to_vec()
     }
 
+    pub fn as_bytes(&self) -> &[u8] {
+        self.as_ref()
+    }
```

Comment:
Agreed with the above, seems like it might affect internal integrators more than Scrypto developers. I think not even the toolkit relied on `as_ref` we typically relied on `node_id.0` for this.